doc/releases/changelog-dev.md

```diff
@@ -471,4 +474,4 @@ Mudit Pandey,
 Alex Preciado,
 Matthew Silverman,
 Jay Soni.
-
+Jacky Jiang,
```

Comment:
```suggestion
```

Reply:
Would you mind moving your name up to below Soran? (alphabetical order)

Reply:
Yup, done =)